src/Runtime/Plugin.cs

```diff
@@ -7,7 +7,7 @@ namespace DiscoAPI.Runtime;
 [BepInPlugin(
     GUID,
     "Disco Elysium Modding API",
-    "0.0.1"
+    "0.0.1-indev"

```

Comment:
this disagrees with the version defined in the `csproj` (`0.1.0`). i don't have an opinion on when we choose to increment version numbers (we're not in a place to be imposing semver or anything), but they should probably be consistent. NB: both corresponding locations in the preload say `0.1.0` so probably:
```suggestion
    "0.1.0-indev"
```

Reply:
Thanks for spotting this